### DISCUSSION TITLE
Generate Helm charts manifests with Quarkus Helm

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#29034] [Quarkus Blog/Website]
# Generate Helm charts manifests with Quarkus Helm

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-helm/

**Comment by adriens** [comment]
Hi, looks like this extension is going to save us a tons of time : would you share some code like GH Ation that shows the CI to push the chart to a registr, let's say any ChartMuseum for example ?

**Reply by adriens** [reply]
cf 
- https://github.com/quarkiverse/quarkus-helm/issues/371
